libcontainer/configs/cgroup_unix.go

```diff
@@ -28,11 +28,14 @@ type Cgroup struct {
 
 type Resources struct {
 	// If this is true allow access to any kind of device within the container.  If false, allow access only to devices explicitly listed in the allowed_devices list.
-	AllowAllDevices bool `json:"allow_all_devices"`
-
-	AllowedDevices []*Device `json:"allowed_devices"`
-
-	DeniedDevices []*Device `json:"denied_devices"`
+	// Deprecated
```

Comment:
Given we're still pre-v1.0 - is there any reason to keep these around instead of just removing them immediately?